src/openforms/registrations/contrib/demo/plugin.py

```diff
@@ -26,6 +26,9 @@ def get_reference_from_result(self, result: None) -> NoReturn:
     def update_payment_status(self, submission: "Submission"):
         print(submission)
 
+    def test_config(self) -> None:
+        print("test plugin")
```

Comment:
Well... to make te config page do something, it should return `True` (since this one always works)